scripts/build.sh

```diff
@@ -19,20 +19,8 @@ set -eu
 # Move into the top-level directory of the project.
 cd "$(dirname "${BASH_SOURCE[0]}")/.." > /dev/null
 
-# Only regenerate the .js and .t.ds file if the protos have changed (i.e.,
-# it's a fresh checkout or update_protos.sh has been executed again and
-# deleted the old generated files). This shaves several seconds off the
-# extension's build time.
-if [[ ! -f src/protos/protos.js ]] ; then
-  sed -e "s#^#src/protos/#" src/protos/protos_list.txt | \
-      xargs pbjs -t static-module -o src/protos/protos.js
-fi
-if [[ ! -f src/protos/protos.d.ts ]] ; then
-  pbts -o src/protos/protos.d.ts src/protos/protos.js
-fi
+node esbuild-debug-adapter.js
+node esbuild.js
 
-# Convert yaml language definition to json form requred by vscode.
-js-yaml syntaxes/bazelrc.tmLanguage.yaml > syntaxes/bazelrc.tmLanguage.json
-
-# Compile the rest of the project.
+# Compile the rest of the project for development flow like testing.
 tsc "$@" -p ./
```

Comment:
Why do you still need `tsc` here?  `esbuild` handles compilation.  Running `tsc` remains useful for testing but not so much for build.